HuntBuddy/Windows/ConfigurationWindow.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Numerics;
+using Dalamud.Interface.Windowing;
+using ImGuiNET;
+
+namespace HuntBuddy.Windows;
+
+/// <summary>
+/// Configuration window.
+/// </summary>
+public class ConfigurationWindow : Window
+{
+    public ConfigurationWindow() : base(
+        $"{Plugin.Instance.Name} configuration",
+        ImGuiWindowFlags.NoDocking,
+        true)
+    {
+        this.Size = Vector2.Zero;
+        this.SizeCondition = ImGuiCond.Always;
+        
+        if (Plugin.Instance.Configuration.LockWindowPositions)
```

Comment:
This needs to go in the pre-draw function, or else it'll only ever set the flags when the plugin first loads and changing the settings won't have any effect live.

Reply:
Addressed in 93872f1